app/templates/home.hbs

```diff
@@ -1,50 +1,51 @@
-<ContentHeader />
+<AuContentHeader
+  @titlePartOne="We geven je alle beslissingen"
+  @titlePartTwo="uit JOUW gemeente"
+>
+  {{!-- <img
+    sizes="50vw"
+    src={{"/assets/loket-header-1600.jpg"}}
+    alt="Foto van een laptop met daarop het vlaanderen logo."
+  /> --}}
+</AuContentHeader>
+<AuBodyContainer
+  style="display: flex; justify-content: center; align-items: center;"
+>
 
-<div class="au-u-flex" style="height: 100%;">
-  <SearchSidebar
-    @municipalityPlaceholder="Selecteer een optie"
-    @municipalityOnChange={{this.handleMunicipalityChange}}
-    @municipalityOptions={{@model}}
-    @municipalityAllowClear={{true}}
-    @keywordPlaceholder={{"Terrasvergunning"}}
-    @keywordOnChange={{this.handleKeywordChange}}
-    @applyDatePicker={{this.applyDatePicker}}
-    @cancelDatePicker={{this.cancelDatePicker}}
-    @hideDatePicker={{this.hideDatePicker}}
-    @valueMunicipality={{this.selectedMunicipality}}
-    @valueKeyword={{this.keyword}}
-    @valueStartDate={{this.plannedStartMin}}
-    @valueEndDate={{this.plannedStartMax}}
-  />
-
-  <section
-    class="au-u-flex au-u-flex--column"
-    style="width: 100%; height: 100%;"
-  >
-    <AuFormRow
-      @alignment="inline"
-      class="au-u-margin-bottom au-u-margin-top au-u-padding-right-large au-u-padding-left-large"
-    >
-      <section style="margin-left: auto;" class="au-u-flex">
-        <AuLabel for="sort-on" style="width: 80px; white-space: nowrap;">
-          Sorteer op
-        </AuLabel>
-        <select
-          type="select"
-          onchange={{action "handleSort"}}
-          class="au-c-input"
-          style="width: 200px;"
-        >
-          <option>Relevantie</option>
-          <option>Datum</option>
-        </select>
-      </section>
-    </AuFormRow>
-    <div
-      class="au-u-padding-left-large au-u-padding-right-large au-u-flex au-u-flex--column"
-      style="overflow-y: scroll; height: auto; padding-bottom:10vh; scroll-behavior: smooth;"
+  <AuFormRow style="width: 40%;">
+    <AuLabel
+      for={{"municipality"}}
+      class="au-u-h4 au-u-margin-bottom-small"
+      @inline={{true}}
     >
-      {{outlet}}
-    </div>
-  </section>
-</div>
+      In welke gemeente(s) wil je zoeken?
+    </AuLabel>
+    <section class="au-u-flex" style="width: 100%;">
+      <PowerSelect
+        style="width: 80%;"
+        id="municipality"
+        @onChange={{this.handleMunicipalityChange}}
+        @selected={{this.selectedMunicipality}}
+        @options={{@model.municipalities}}
+        @allowClear={{true}}
+        @searchField="label"
+        @placeholder={{"Alle gemeentes"}}
+        @noMatchesMessage={{"Geen gemeente gevonden"}}
+        @searchMessage="Aan het zoeken..."
+        @searchEnabled={{true}}
+        class="au-u-margin-right-small"
+        as |singleSelect|
+      >
+        {{singleSelect.label}}
+      </PowerSelect>
+      <AuButton
```

Comment:
The button has to be disabled when municipality is empty, or at least do not add empty filter in url `gemeentes=`. 
![image](https://github.com/lblod/frontend-burgernabije-besluitendatabank/assets/3050307/c611967b-946f-4183-95c3-0f9622edb379)